modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -12126,18 +12132,46 @@ type TimeChargeNoDataDiscount implements TimeChargeDiscount {
   interval: TimestampInterval!
 
   "Time amount to discount from the program."
-  program: TimeSpan!
+  program: TimeSpan! @deprecated(reason: "renamed to 'amount'")
```

Comment:
Do we want to deprecate fields? I thought at this stage we'd just delete them

Reply:
In this case it was trivial and doesn't leave (much) cruft so I just deprecated.  Might be that nobody uses it anyway.